coderd/workspaces_test.go

```diff
@@ -550,19 +550,16 @@ func TestWorkspaceUpdateAutostart(t *testing.T) {
 			name:          "invalid location",
 			schedule:      ptr.Ref("CRON_TZ=Imaginary/Place 30 9 * * 1-5"),
 			expectedError: "parse schedule: provided bad location Imaginary/Place: unknown time zone Imaginary/Place",
-			// expectedError: "status code 500: Invalid autostart schedule\n\tError: parse schedule: provided bad location Imaginary/Place: unknown time zone Imaginary/Place",
```

Comment:
**Nit**: Are these leftover debug?

Reply:
Yep, left over from some previous PR where they snuck in!